keras_cv/models/efficientnet_lite_test.py

```diff
@@ -56,6 +57,17 @@ def test_application_variable_input_channels(self, app, last_dim, args):
     def test_model_can_be_used_as_backbone(self, app, last_dim, args):
         super()._test_model_can_be_used_as_backbone(app, last_dim, args)
 
+    @parameterized.parameters(*MODEL_LIST)
```

Comment:
Please remove this change for now so that we can keep all of these serialization changes in one PR in #1389.

Reply:
Done.